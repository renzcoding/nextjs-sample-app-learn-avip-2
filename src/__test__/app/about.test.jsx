import AboutPage from "@/app/about/page";
import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import { describe, it } from "node:test";

describe("AboutPage", () => {
  it("should render", () => {
    // const heading = screen.getByRole("heading", { level: 1 });
    // expect(heading).toBeInTheDocument();

    const { container } = render(<AboutPage />);
    expect(container).toMatchSnapshot();
  });
});
