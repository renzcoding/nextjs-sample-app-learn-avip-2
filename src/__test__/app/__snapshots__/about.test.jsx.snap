// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  <h1>
    About Page
  </h1>
</div>
`;
